appveyor.yml

```diff
@@ -0,0 +1,28 @@
+build: off
+
+os: unstable
+
+platform:
+  - x86
+  - x64
+
+branches:
+  only:
+    - master
+
+skip_tags: true
```

Comment:
we should also run it against tags as well since we'll be using them for releases